track_local_static.go

```diff
@@ -0,0 +1,235 @@
+// +build !js
+
+package webrtc
+
+import (
+	"io"
+	"strings"
+	"sync"
+	"sync/atomic"
+
+	"github.com/pion/rtp"
+	"github.com/pion/rtp/codecs"
+	"github.com/pion/webrtc/v3/internal/util"
+	"github.com/pion/webrtc/v3/pkg/media"
+)
+
+// trackBinding is a single bind for a Track
+// Bind can be called multiple times, this stores the
+// result for a single bind call so that it can be used when writing
+type trackBinding struct {
+	id          string
+	ssrc        SSRC
+	payloadType PayloadType
+	writeStream TrackLocalWriter
+}
+
+// TrackLocalStaticRTP  is a TrackLocal that has a pre-set codec and accepts RTP Packets.
+// If you wish to send a media.Sample use TrackLocalStaticSample
+type TrackLocalStaticRTP struct {
```

Comment:
Maybe it's better to be a subpackage?

Reply:
That was really hard because of testing. I had to move a lot of tests to the subpackage, but then I still wanted to touch WebRTC internals.

We can do this if you feel strongly though!

Reply:
It sounds ok to be here, but as a general functionality in the main package, parameters like MTU should be able to be configured.